klaviyo/data_privacy.py

```diff
@@ -0,0 +1,36 @@
+from .api_helper import KlaviyoAPI
+from .exceptions import KlaviyoException
+
+class DataPrivacy(KlaviyoAPI):
+    DATA_PRIVACY = 'data-privacy'
+    DELETION_REQUEST = 'deletion-request'
+    ALLOWED_ID_TYPES = [
+        'email',
+        'phone_number',
+        'person_id',
+    ]
```

Comment:
Integrations has been espousing enums, it'd be cool if we wanted to use one here. Not necessary but just a thought.